src/Components/App/index.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import NavBar from "../Navbar/NavBar";
+
```

Comment:
@Justin-ml Please format the Code Before Commiting.

Reply:
Done.